### ISSUE TITLE
200MB Limit

### ISSUE THREAD

**Issue by MidnightKittenCat** [abdeladim-s/subsai#74]
"File must be 200.0MB or smaller." when inputting a file over 200mb, I was hopping it can be transcribed/merged without showing the video if >200mb as I would assume thats the main issue, that the library being used can only shows videos <200mb, if this is the case can we still have it so we can download the translated/transcribed video at the end without the preview?

**Comment by MidnightKittenCat** [comment]
Using "subsai-webui --server.maxUploadSize 5000" seems to bypass this limit allowing for 5gb and up if you change it, next issue is rocm is now incompatible again with one of the recent updates.

**Comment by abdeladim-s** [comment]
@MidnightKittenCat, I tried to catch the error if the file size exceeds the size limit, and it is working fine on my side with a 300 MB file, it can be transcribed without any errors. You can try the last commit. 

But I would highly suggest to not use large media files with the webui, as streamlit is limited, use the command line interface for large media files instead.

**Comment by MidnightKittenCat** [comment]
> @MidnightKittenCat, I tried to catch the error if the file size exceeds the size limit, and it is working fine on my side with a 300 MB file, it can be transcribed without any errors. You can try the last commit.
> 
> But I would highly suggest to not use large media files with the webui, as streamlit is limited, use the command line interface for large media files instead.

Is the issue with the fact its trying to display the video? If so wouldn’t we just be able to disable it?

and would you be able to give a brief example of how to translate using faster-whisper, with auto detect language to en? Thanks!

**Comment by abdeladim-s** [comment]
No, if you tried the latest update and didn't work then I don't think the issue is related to the video because the video preview is disabled now if the size exceeds the limit. 
Could you please share the file you are using with me, I will give it a try on my end.  

The command line is easy to use, you can find how to use it on the [Readme page](https://github.com/abdeladim-s/subsai#cli).

**Comment by MidnightKittenCat** [comment]
> No, if you tried the latest update and didn't work then I don't think the issue is related to the video because the video preview is disabled now if the size exceeds the limit. Could you please share the file you are using with me, I will give it a try on my end.
> 
> The command line is easy to use, you can find how to use it on the [Readme page](https://github.com/abdeladim-s/subsai#cli).

https://privnote.com/GBV341jj#EVzSe0krY

priv noted for obvious reasons. The link is inside it, the privnote will self destruct after being read.

**Comment by MidnightKittenCat** [comment]
If you have any issues with downloading it, let me know and I’ll directly upload it.

**Comment by abdeladim-s** [comment]
> > No, if you tried the latest update and didn't work then I don't think the issue is related to the video because the video preview is disabled now if the size exceeds the limit. Could you please share the file you are using with me, I will give it a try on my end.
> > The command line is easy to use, you can find how to use it on the [Readme page](https://github.com/abdeladim-s/subsai#cli).
> 
> https://privnote.com/GBV341jj#EVzSe0krY
> 
> priv noted for obvious reasons. The link is inside it, the privnote will self destruct after being read.

Yeah I understand, I couldn't download the file. but no problem, I tried with a movie, almost 900 MB, it worked without any issues. So, you may need to investigate more on your end. 

Also if you are using a language other than English (Japanese in your case) you will need to use the large model, 'casue it is the only one which is Multilingual.

**Comment by MidnightKittenCat** [comment]
![image](https://github.com/abdeladim-s/subsai/assets/66114416/a3cbd37b-bd0a-4267-a5cb-d6f6a1ef13e3)

Seems to translate, but also it just spams "I'm sorry."

**Comment by abdeladim-s** [comment]
@MidnightKittenCat, this seems like a model issue, Please try different models and different parameters.
